pallets/parachain-staking/src/mock.rs

```diff
@@ -189,6 +189,11 @@ impl ExtBuilder {
 		self
 	}
 
+	/// add a delegation by providing a Vec of tuples where each tuple is a delegation expressing:
+	/// 1. delegator's AccountId
+	/// 2. collator's AccountId
+	/// 3. delegation amount
+	/// (Note that this is passed directly to GenesisConfig's build())
```

Comment:
This is the genesis config format, these docs would be better there.

I also think it could be expressed more succinctly ( instead of listing each item on different lines): "`Vec` of tuples of the format `(delegator AccountId, collator AccountId, delegation Amount)`"

Reply:
85a0299